src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkIntegrationTest.java

```diff
@@ -119,16 +122,29 @@ private String getCloudInputs() {
         };
     }
 
+    @Test(dataProvider = "BQSRTest")
```

Comment:
`groups = "Spark"` for consistency with https://github.com/broadinstitute/gatk/pull/1814